### PR TITLE
beta version of Gradle exporter

### PR DESCRIPTION
**Pull request by bissquit**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM python:3.7-slim
```
Comment (rvadim) [inline Dockerfile]: 3.8 or 3.9 is better

File: gradle_exporter.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
+import os
+import json
+import aiohttp
+from aiohttp import web
+import asyncio
+import logging
+import argparse
+
+logging.basicConfig(level=os.getenv("LOG_LEVEL", logging.DEBUG), format='%(asctime)s - %(levelname)s - %(message)s')
```
Comment (rvadim) [inline gradle_exporter.py]: It is better to use `loggin.INFO` by default

**Comment by rvadim** [comment]
Gradle - build tool, CLI
Gradle (Enterprise) Server - Web application for store and analysis builds
So it is better to named exporter like gradle-server-exporter or somthing similar.

**Review by rvadim — APPROVED** [review]

**Comment by rvadim** [comment]
I would recommend you to not use Asyncio for this kind of tasks

File: gradle_exporter.py (hunk 2 above)
Comment (bissquit) [reply gradle_exporter.py]: Fixed

File: Dockerfile (hunk 1 above)
Comment (bissquit) [reply Dockerfile]: Done

**Pull request merged**